tfjs-backend-webgpu/src/max_pool_backprop_webgpu.ts

```diff
@@ -91,3 +91,89 @@ export class MaxPool2DBackpropProgram implements WebGPUProgram {
     return userCode;
   }
 }
+
+export class MaxPool3DBackpropProgram implements WebGPUProgram {
+  outputShape: number[];
+  shaderKey: string;
+  dispatchLayout: {x: number[]};
+  dispatch: [number, number, number];
+  variableNames = ['dy', 'maxPos'];
+  uniforms =
+      `strides : vec3<i32>, pads : vec3<i32>, dilations : vec3<i32>, filterDims : vec3<i32>,
```

Comment:
nit: dilations should be removed?

Reply:
Done. Had forgotten to remove `dilations` information for MaxPool3DGrad kernel. Thanks.